pod.go

```diff
@@ -64,6 +64,9 @@ type State struct {
 
 	// File system of the rootfs incase it is block device
 	Fstype string `json:"fstype"`
+
+	// Bool to indicate if the drive for a container was hotplugged.
+	IsDriveHotplugged bool `json:"isDriveHotplugged"`
```

Comment:
This being a variable and not a function, I would call it `HotpluggedDrive` instead.

Reply:
Fixed